bats/sql-diff.bats

```diff
@@ -482,6 +482,7 @@ SQL
 
     dolt diff --sql newbranch firstbranch > query
     dolt checkout firstbranch
+    skip "add + drop doesn't work, we have to track renames"
```

Comment:
Unique tags will function as table tags and allow SQL diff to generate `RENAME` statements

Reply:
K cool.